channeld/full_channel.c

```diff
@@ -414,7 +414,7 @@ static enum channel_add_err add_htlc(struct channel *channel,
 			- commit_tx_num_untrimmed(removing, feerate, dust,
 						  recipient);
 
-		fee_msat = commit_tx_base_fee(feerate, untrimmed);
+		fee_msat = commit_tx_base_fee(feerate, untrimmed) * 1000;
```

Comment:
should we annotate this method to denote the units it returns, ie `commit_tx_base_fee_sat`